nixos/modules/config/users-groups.nix

```diff
@@ -35,7 +35,7 @@ let
   '';
 
   hashedPasswordDescription = ''
-    To generate a hashed password run `mkpasswd -m sha-512`.
+    To generate a hashed password run `mkpasswd -m yescrypt`.
```

Comment:
Why specify yescrypt explicitly over just using the default (which is yescrypt as of now), consistency or something (I mean in case the default changes)?

Reply:
No strong opinion as mentioned in the old PR. We can drop the `-m` parameter completely if that's what everyone prefers.

Reply:
Removed the explicit method parameter.